# Chronic Kidney Disease in Florida
Here we will be investigating chronic kidney disease in Florida. The data being used is publicly available at CMS website.

In [1]:
# Importing libraries
import pandas as pd
import missingno as msno
import seaborn as sns
import matplotlib as plt
pd.set_option('display.max_rows', 200) # Setting wider view

In [2]:
# Reading in original CSV
df = pd.read_csv('C:/Users/Bader Ale/Desktop/CC_R20_P08_v10_D18_WWDSE_Cond.csv')

In [3]:
# Viewing first 10 records in dataset
df.head(10)

Bene_Geo_Lvl Bene_Geo_Desc  Bene_Geo_Cd Bene_Age_Lvl Bene_Demo_Lvl  \
0        State       Alabama          1.0          All           All   
1        State       Alabama          1.0          65+   Dual Status   
2        State       Alabama          1.0          <65   Dual Status   
3        State       Alabama          1.0          All   Dual Status   
4        State       Alabama          1.0          65+   Dual Status   
5        State       Alabama          1.0          <65   Dual Status   
6        State       Alabama          1.0          All   Dual Status   
7        State       Alabama          1.0          65+           Sex   
8        State       Alabama          1.0          <65           Sex   
9        State       Alabama          1.0          All           Sex   

          Bene_Demo_Desc      Bene_Cond  Prvlnc  Tot_Mdcr_Stdzd_Pymt_PC  \
0                    All  Alcohol Abuse  0.0188              25102.3405   
1          Medicare Only  Alcohol Abuse  0.0118                     NaN   
2          Medicare Only  Alcohol Abuse  0.0320                     NaN   
3          Medicare Only  Alcohol Abuse  0.0147                     NaN   
4  Medicare and Medicaid  Alcohol Abuse  0.0238                     NaN   
5  Medicare and Medicaid  Alcohol Abuse  0.0465                     NaN   
6  Medicare and Medicaid  Alcohol Abuse  0.0347                     NaN   
7                 Female  Alcohol Abuse  0.0058                     NaN   
8                 Female  Alcohol Abuse  0.0235                     NaN   
9                 Female  Alcohol Abuse  0.0092                     NaN   

   Tot_Mdcr_Pymt_PC  Hosp_Readmsn_Rate  ER_Visits_Per_1000_Benes  
0        23348.6039             0.2413                 2184.7557  
1               NaN                NaN                       NaN  
2               NaN                NaN                       NaN  
3               NaN                NaN                       NaN  
4               NaN                NaN                       NaN  
5               NaN                NaN                       NaN  
6               NaN                NaN                       NaN  
7               NaN                NaN                       NaN  
8               NaN                NaN                       NaN  
9               NaN                NaN                       NaN

In [4]:
# Selecting only Florida and saving as new dataframe
df_florida= df[df['Bene_Geo_Desc'] == 'Florida']

In [5]:
# Checking the dimensionality of df_florida
df_florida.shape

(630, 12)

Dataframe _df_florida_ has a total of 12 columns and 630 records.

Let's now check for any duplicated values.

In [6]:
# Checking for duplicated values
df_florida.duplicated().value_counts()

False    630
dtype: int64

No duplicated values found. The next step is to address missing values.

In [ ]:
# Checking for missing values in Florida dataframe
df_florida.isnull().sum()

In [ ]:
df_florida['Bene_Cond'].value_counts()

In [ ]:
df_fl_ckd = df_florida[df_florida['Bene_Cond'] == "Chronic Kidney Disease"]

In [ ]:
df_fl_ckd.shape

In [ ]:
df_fl_ckd

In [ ]:
df_fl_ckd.loc[df_fl_ckd.duplicated()]


In [ ]:
df_fl_ckd['Prvlnc'] \
    .plot(kind='kde')
